azure-sfmesh-maven-plugin/src/main/java/com/microsoft/azure/maven/servicefabric/Utils.java

```diff
@@ -0,0 +1,216 @@
+            }
+            return stdout;
+        } catch (IOException e){
+            logger.error(e);
+            throw new MojoFailureException(String.format("Error while " +
+                "running the %s command", command));
+        } catch (InterruptedException e) {
+            logger.error(e);
+            throw new MojoFailureException(String.format("Interrupted while " +
+                "running command %s", command));
+        }
+    }
+
+    public static String executeCommand(Log logger, String[] command) throws MojoFailureException{
+        try {
+            logger.info(String.format("Executing command %s", Arrays.toString(command)));
+            final Process p;
+            p = Runtime.getRuntime().exec(command);
+            p.waitFor();
+            final int exitCode = p.exitValue();
+            final String stderr = IOUtil.toString(p.getErrorStream(), "UTF-8");
+            final String stdout = IOUtil.toString(p.getInputStream(), "UTF-8");
+            logger.debug(String.format("STDOUT: %s", stdout));
+            if (stderr != null && stderr.length() > 0){
+                if (exitCode != 0){
+                    logger.error(String.format("Process exited with exit code %d", exitCode));
+                    logger.error(String.format("STDERR: %s", stderr));
+                    throw new MojoFailureException(String.format("Error" +
+                        "while running the %s command", Arrays.toString(command)));
+                } else {
+                    logger.info(String.format("STDERR: %s", stderr));
+                }
+            }
+            return stdout;
+        } catch (IOException e){
+            logger.error(e);
+            throw new MojoFailureException(String.format("Error while " +
+                "running the %s command", Arrays.toString(command)));
+        } catch (InterruptedException e) {
+            logger.error(e);
+            throw new MojoFailureException(String.format("Interrupted while " +
+                "running command %s", Arrays.toString(command)));
+        }
+    }
+
+    public static void checksfctlinstallation(Log logger) throws MojoFailureException{
+        if (Utils.isWindows()){
+            Utils.executeCommand(logger, "sfctl --help  > NUL 2>&1");
+        } else {
+            Utils.executeCommand(logger, "sfctl --help >> /dev/null 2>&1");
+        }
+    }
+
+    public static void checkazinstallation(Log logger) throws MojoFailureException{
+        if (Utils.isWindows()){
+            Utils.executeCommand(logger, "az mesh --help > NUL 2>&1");
+        } else {
+            Utils.executeCommand(logger, "az mesh --help >> /dev/null 2>&1");
+        }
+    }
+
+    public static void connecttounsecurecluster(Log logger,
+        String endpoint) throws MojoFailureException{
+        Utils.executeCommand(logger, "sfctl cluster select --endpoint " + endpoint);
+    }
+
+    public static void connecttosecurecluster(Log logger, String endpoint,

```

Comment:
nit: We should use `CamelCase` for method name

Reply:
fixed